### ISSUE TITLE
Incompatible with Python<3.10

### ISSUE THREAD

**Issue by aidanmelen** [abrahammurciano/discord-lumberjack#31]
I am getting a stack trace when I try to import the latest version. Please see steps below to reproduce.

```bash
$ pip install discord-lumberjack
$ pip list | grep discord-lumberjack
discord-lumberjack 1.0.2

$ python
>>> import discord_lumberjack
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/__init__.py", line 8, in <module>
    from . import handlers
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/handlers/__init__.py", line 9, in <module>
    from .discord_handler import DiscordHandler
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/handlers/discord_handler.py", line 6, in <module>
    from discord_lumberjack.message_creators import BasicMessageCreator, MessageCreator
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/message_creators/__init__.py", line 11, in <module>
    from .embed_message_creator import EmbedMessageCreator
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/message_creators/embed_message_creator.py", line 5, in <module>
    from .embed import Embed, EmbedFieldSetter, embed_length, empty_embed
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/message_creators/embed.py", line 89, in <module>
    class EmbedFieldSetter:
  File "/root/.cache/pypoetry/virtualenvs/bobcat-miner-BajaH40D-py3.9/lib/python3.9/site-packages/discord_lumberjack/message_creators/embed.py", line 100, in EmbedFieldSetter
    key_chain: Sequence[str | int],
TypeError: unsupported operand type(s) for |: 'type' and 'type'
>>> import discord-lumberjack
KeyboardInterrupt
```

:boom: :boom: :boom:

**Comment by abrahammurciano** [comment]
You need python 3.10

**Comment by abrahammurciano** [comment]
I believe I could make it compatible with earlier python versions, I'll consider it

**Comment by abrahammurciano** [comment]
Just released v1.0.3 which is compatible with python 3.8 and up

**Comment by aidanmelen** [comment]
thanks a bunch!
